WebRoot/messages/ZhMsg.properties

```diff
@@ -1854,6 +1854,7 @@ account.IDENTITY_EXISTS = Identity already exists.
 account.TOO_MANY_IDENTITIES = An identity could not be created because you have exceeded the limit.
 account.DATA_SOURCE_EXISTS = Data source already exists.
 account.TOO_MANY_DATA_SOURCES = A data source could not be created because you have exceeded the limit.
+account.NO_SHARE_EXISTS = No share exists.
```

Comment:
Sure. Created ticket to track progress for now.
https://jira.corp.synacor.com/browse/ZCS-1953

Thanks